scripts/amd/gemm/matmul_kernel.py

```diff
@@ -41,8 +42,12 @@ def matmul_kernel(
     acc_dtype = tl.float32 if a_ptr.type.element_ty != tl.int8 else tl.int32
     accumulator = tl.zeros((BLOCK_SIZE_M, BLOCK_SIZE_N), dtype=acc_dtype)
     for k in range(0, tl.cdiv(K, BLOCK_SIZE_K * SPLIT_K)):
-        a = tl.load(a_ptrs)
-        b = tl.load(b_ptrs)
+        if EVEN_K:
```

Comment:
For line 35 and 36, maybe we can add the module of `M` and `N` of the offset added to `a_ptr` and `b_ptr` to support random M and N values.

Reply:
We can, but that will harm performance a lot.
Let me create a ticket for this issue.